Resources/config/sonata_admin.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="app.admin.logs" alias="Ekino\DataProtectionBundle\Admin\LogsAdmin" />
+        <service id="Ekino\DataProtectionBundle\Admin\LogsAdmin">
+            <argument>Ekino\DataProtectionBundle\Admin\LogsAdmin</argument>
```

Comment:
I think it should be empty

Reply:
why should it be empty? it's the admin code to potentially retrieve the admin through the pool